doc/architecture/environmental_effect_model.adoc

```diff
@@ -0,0 +1,9 @@
+= Environmental effect model
+
+Environmental effect models consume sensor-view data and produce sensor-view data.
+Environmental effect models may, for example, include the following:
+
+* Shadows and oclusion
```

Comment:
```suggestion
* Shadows and occlusion
```

Reply:
Maybe Shadows and occlusions (since we used plural for shadows than to stay consistent, also use it for occlusions)

Reply:
Fixed typo and switched to plural, see https://github.com/OpenSimulationInterface/open-simulation-interface/commit/8a1488480f5f4e300b9ff3f57c4a672c00c86b4c.